pixi/src/checks/SafeBrowsingCheck.js

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const API_ENDPOINT =
+  'https://safebrowsing.googleapis.com/v4/threatMatches:find?key=';
+const API_KEY = 'AIzaSyCKKBvhpC73FqDcO-T7_4Yqdx4nQXh2sQY';
+const API_URL = `${API_ENDPOINT}${API_KEY}`;
+
+export default class SafeBrowsingCheck {
+  async run(pageUrl) {
+    try {
+      const apiResult = await this.fetchJson(pageUrl);
+      console.log(apiResult);
+      return this.createReportData(apiResult);
+    } catch (e) {
+      // TODO: Create an Error Report
```

Comment:
This is done in `BasicCheckReport` (?)

Reply:
Hey @caroqliu, the structure of how the error reports will be rendered is now implemented.  Let me know if you have any questions/ideas. :)